src/move/tasks/analyze_latent.py

```diff
@@ -60,9 +61,14 @@ def analyze_latent(config: MOVEConfig):
             model=model,
             train_dataloader=train_dataloader,
         )
+        losses = output[:-1]
         torch.save(model.state_dict(), model_path)
         logger.debug("Generating plot: loss curves")
-        fig = viz.plot_loss_curves(output[:-1])
-        fig.savefig(str(output_path / "loss_curve.png"), bbox_inches="tight")
-
+        loss_fig = viz.plot_loss_curves(losses)
+        loss_fig.savefig(str(output_path / "loss_curve.png"), bbox_inches="tight")
```

Comment:
does this expect a `str`?

Reply:
The type hints of `matplotlib`'s `savefig` are set incorrectly. It is marked to expect a string or a file object. A `Path` works, but then the type checker complains about it not being supported. I decided to just convert the input to string to get rid of the warnings from the type checker.